mpa/modules/models/heads/supcon_cls_head.py

```diff
@@ -57,7 +72,21 @@ def __init__(self, num_classes: int, in_channels: int, aux_mlp, loss, aux_loss,
         else:
             self.aux_mlp = nn.Linear(in_features=in_channels, out_features=out_channels)
 
-    def forward_train(self, x, gt_label):
+    def simple_test(self, img: Tensor):
+        """
+        Test without data augmentation.
+        """
+        cls_score = self.classifier(img)
+
+        if isinstance(cls_score, list):
+            cls_score = sum(cls_score) / float(len(cls_score))
+        pred = F.softmax(cls_score, dim=1) if cls_score is not None else None
+        if torch.onnx.is_in_onnx_export():
+            return pred
```

Comment:
Currently, in classification task we export model without activations (for backward compatibility with the legacy code). Please change it for consistency.

Reply:
See https://github.com/openvinotoolkit/model_preparation_algorithm/pull/83